.travis.yml

```diff
@@ -0,0 +1,23 @@
+sudo: required
+
+services:
+  - docker
+
+script:
+  - make test-ci
+
+after_success:
+  - docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS
+  - export REPO=saltside/await
+  - export TAG=`if [ "$TRAVIS_BRANCH" == "master" ]; then echo "latest"; else echo $TRAVIS_BRANCH ; fi`
+  - docker build -f Dockerfile -t $REPO:$COMMIT .
+  - docker tag $REPO:$COMMIT $REPO:$TAG
+  - docker tag $REPO:$COMMIT $REPO:travis-$TRAVIS_BUILD_NUMBER
```

Comment:
This was using the practices suggested by the Travis CI community, but as we discussed I will replace the whole tagging scheme with the major, minor, patch logic.